### ISSUE TITLE
[1.15] Exchanging Gadget seems to silk touch blocks by default

### ISSUE THREAD

**Issue by Merlo17** [Direwolf20-MC/BuildingGadgets#487]
* Forge version: 31.2.20
* Building Gadgets version: 3.3.5

Any exchange made with the gadget will act as silk touch, even when not enchanted with silk touch. I'm pretty sure this is not intended behaviour.

**Comment by MichaelHillcox** [comment]
This is true, this is not intended and don't tell dire or we'll get told off, joking of course haha. But yeah, this has been an issue since the 1.14 port so I'll lable and assign this one to make sure it's resolved as it's a stupid oversight.
